public/app/plugins/panel/timeseries/plugins/ThresholdControlsPlugin.tsx

```diff
@@ -53,16 +53,26 @@ export const ThresholdControlsPlugin: React.FC<ThresholdControlsPluginProps> = (
       if (Number.isNaN(yPos) || !Number.isFinite(yPos)) {
         continue;
       }
-      if (yPos < 0 || yPos > plot.bbox.height / window.devicePixelRatio) {
-        continue;
+
+      let handleYPos = yPos;
+      const height = plot.bbox.height / window.devicePixelRatio;
+
+      // the threshold is below the visible series area
+      if (yPos < 0) {
+        handleYPos = 0;
+      }
+
+      // the threshold is above the visible series area
+      if (yPos > height) {
+        handleYPos = height;
```

Comment:
Got the idea, but wonder if this won't create even more confusion with the threshold positioned against the wrong y-value. Maybe the handle should also have some kind of indicator that it's rendereder for a value that's outside the bound? I.e. dimmed handle plus indicator icon? I see unicons have these icons that would >possibly< work:

![image](https://user-images.githubusercontent.com/2376619/141771308-16fe5dc7-f18d-4ed2-b786-2acec9f0e406.png)
![image](https://user-images.githubusercontent.com/2376619/141771398-9039d8cc-3fcd-4984-9f09-3481e6c72d30.png)

![image](https://user-images.githubusercontent.com/2376619/141771546-61f5b97b-dde8-4915-80c0-6a112c2eeedd.png)


WDYT @peterholmberg ?

Reply:
I was thinking about this as well, a visual cue to indicate it's out-of-bounds would help to make it more clear 👍

Reply:
I tried a few different ones, so far I prefer the `angle-down` and `angle-up`. <img width="342" alt="Screenshot 2021-11-15 at 18 34 51" src="https://user-images.githubusercontent.com/868844/141828079-3052f5bc-aa6c-4c14-aac1-664ada83eb77.png">
<img width="342" alt="Screenshot 2021-11-15 at 18 31 53" src="https://user-images.githubusercontent.com/868844/141828084-80dc6cfe-9532-496f-938f-5ea073ef2bb7.png">
<img width="342" alt="Screenshot 2021-11-15 at 18 30 11" src="https://user-images.githubusercontent.com/868844/141828088-fa758f18-9c76-4507-abb6-66f3d7773bbe.png">

Reply:
I've updated the PR description with a video of the new behavior and the updated styles